portfolioAstro/src/styles/melo.scss

```
@@ -0,0 +1,484 @@
+$black: #212121;
+$yellow: #fdd835;
+$lightblue: #47d1ff;
+$pink: #ff729f;
+
+$border: 3px solid #212121;
+
+$bottomFifty: 0 auto 50px auto;
```

Stylelint (17.4.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)